tool/cmd/ingest/main.go

```diff
@@ -37,6 +38,7 @@ type metadataCollection struct {
 type args struct {
 	metadataPath string
 	dbPath       string
+	schemaPath   string
```

Comment:
Since I refactored the metadata tool Go binary to live in the `tool` directory, now the DB schema doesn't live alongside it. Instead, pass in the path explicitly as an argument.